content/articles/2018-07-broom-0-5-0.Rmarkdown

```diff
@@ -0,0 +1,127 @@
+- use augment methods on models with matrix covariates specified in a formula, which will error
+
+Augment will error with matrix covariates because tibbles are more strict about their contents than data frames. Data frames allow users to specify columns in a matrix, like so:
+
+```{r}
+y <- 1:3
+x <- matrix(1:9, nrow = 3)
+df <- data.frame(x, y)
+```
+
+Tibbles do not:
+
+```{r, error = TRUE}
+library(tibble)
+
+tibble(x, y)
+```
+
+Modelling functions will occassional create a data frame like this, but since the model frame can't be coerced a tibble method, `augment` will fail:
+
+```{r, error = TRUE}
+fit <- lm(y ~ x, df)
+augment(fit)
+```
+
+In some cases, explicitly passing the original dataset via the `data` argument can resolve this:
+
+```{r}
+augment(fit, data = df)
+```
+
+In any case, support for matrix-columns is on the way in dplyr and in a release cycle or two this won't be an issue.
+
+A number of deprecated tidiers (or tidiers that will be deprecated in the next release) have not been updated to use tibble output. Most notably, this includes tidiers for mixed models. More on this below.
+
+## New infrastructure for testing and documentation
+
+We are heavily invested in making it easier to contribute to broom, and also in making broom behavior more standardized and consistent. To this end, we've written new testing and documentation infrastructure that automates much of these processes.
+
+In broom 0.5.0 these tests mostly ensure tibble output along with one or two other consistency items. However, stricter versions of these tests are under development for future releases.
+
+We have also migrated to a new template based documentation strategy. Now repeated documentation material lives in `roxygen2` templates and can easily added to a new tidy method. For an example of how this works, see [`R/aer-tidiers.R`](https://github.com/alexpghayes/broom/blob/master/R/aer-tidiers.R).
+
+## Mixed models are moving to `broom.mixed`
+
+As broom's popularity has grown, broom has grown to encompass a fairly broad range of models. Dave and I have little to no experience with many of these models, and while we can fix bugs in the tidying code, we are no longer able to determine what constitutes a reasonable summary for reasonable summary for many of these models.
+
+Our intended solution is to split broom into several packages for tidying model objects. broom will provide tidiers for the most common tidiers (and those in `base` and `stats`), and then domain experts will manage domain specific tidying packages. Currently we're working on a spec for all these sub-packages to implement. With any luck this we'll have a well-written spec to accompany the next release. We'd like all of the domain specific tidying packages to eventually live in tidymodels, so that users can load a bunch of tools all at once with `library(tidymodels)`.
+
+Mixed models tidiers have long been a bit of a mess in broom, and a while back [`broom.mixed`](
```

Comment:
* Tidiers for mixed models? (Or Mixed-model tidiers…)
* I'd break that first sentence into two. Period after first broom, and then begin the second with "A while back…"